components/dropzone/index.css

```diff
@@ -144,55 +84,97 @@
+}
+
+.spectrum-DropZone-content {
+	max-inline-size: var(--mod-illustrated-message-vertical-maximum-width);
+}
+
+.spectrum-DropZone-actions {
+	margin-block-start: var(--spectrum-drop-zone-body-to-action);
+	text-align: center;
 }
 
 .spectrum-DropZone-button {
 	box-sizing: border-box;
 	border: none;
-	block-size: var(--mod-drop-zone-content-height, var(--spectrum-drop-zone-content-height));
+	block-size: auto;
+
 	max-inline-size: var(--mod-drop-zone-content-max-width, var(--spectrum-drop-zone-content-max-width));
 	padding-block-start: var(--mod-drop-zone-content-top-to-text, var(--spectrum-drop-zone-content-top-to-text));
 	padding-block-end: var(--mod-drop-zone-content-bottom-to-text, var(--spectrum-drop-zone-content-bottom-to-text));
-	background-color: var(--mod-drop-zone-content-background-color, var(--spectrum-drop-zone-content-background-color));
+	padding-inline: var(--mod-drop-zone-edge-to-text, var(--spectrum-drop-zone-edge-to-text));
+	background-color: var(--highcontrast-drop-zone-button-color, var(--mod-drop-zone-content-background-color, var(--spectrum-drop-zone-content-background-color)));
+	max-block-size: var(--spectrum-drop-zone-component-height);
 
 	font-family: var(--mod-drop-zone-content-font-family, var(--spectrum-drop-zone-content-font-family));
 	font-weight: var(--mod-drop-zone-content-font-weight, var(--spectrum-drop-zone-content-font-weight));
-	font-style: var(--mod-drop-zone-content-font-style, var(--spectrum-drop-zone-content-font-style));
-	line-height: var(--mod-drop-zone-content-line-height, var(--spectrum-drop-zone-content-line-height));
+	line-height: var(--mod-drop-zone-title-line-height, var(--spectrum-drop-zone-title-line-height));
+
+	.spectrum-ActionButton-label {
+		font-size: var(--mod-drop-zone-content-font-size, var(--spectrum-drop-zone-content-font-size));
+	}
 
 	&:hover,
 	&:focus {
-		background-color: var(--mod-drop-zone-content-background-color, var(--spectrum-drop-zone-content-background-color));
+		background-color: var(--highcontrast-drop-zone-button-color, var(--mod-drop-zone-content-background-color, var(--spectrum-drop-zone-content-background-color)));
+	}
+}
+
+/********* Illustrated Message *********/
+.spectrum-DropZone {
+	.spectrum-IllustratedMessage {
+		justify-content: center;
 	}
 }
 
 /********* WHCM *********/
 @media (forced-colors: active) {
 	.spectrum-DropZone {
-		/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- used in illustrated */
-		--highcontrast-drop-zone-illustration-color: CanvasText;
-
 		--highcontrast-drop-zone-border-color-hover: Highlight;
-
-		/* Updated values for a nested illustrated message when in a high contrast state */
-		/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- used in illustrated */
-		--highcontrast-illustrated-message-illustration-color: var(--highcontrast-drop-zone-illustration-color);
+		--highcontrast-drop-zone-button-color: Highlight;
```

Comment:
I did notice that in forced-colors in Chrome the button's looking a little transparent, although I will say it didn't look this transparent when I used AssistivLabs to look at real WHCM:

![Screenshot 2025-02-25 at 2 27 17 PM](https://github.com/user-attachments/assets/80aa9873-29fd-4d76-a48e-b776dfc16ec7)
![Screenshot 2025-02-25 at 2 27 25 PM](https://github.com/user-attachments/assets/b09ee75c-2c69-4de4-9224-a63df74b65b6)

I don't think it's a huge issue and what's here mirrors what's in action button, I think, but you could look at whether ButtonText or ButtonFace might work better?